doc/troubleshooting.md

```diff
@@ -83,4 +83,43 @@ To make it work, the cluster admin will need to delete the lock by running
 kubectl delete configmap pravega-operator-lock
 ```
 After that, the new Operator pod will become the leader. If the node comes up later, the extra Operator pod will
-be deleted by Deployment controller. 
\ No newline at end of file
+be deleted by Deployment controller. 
+
+## External-IP details truncated in older Kubectl Client Versions
+
+If we enable external access while deploying a Pravega Cluster, an external IP is assigned to the Pravega Controller endpoint, which is used by users to access it. These external IP details can be viewed in the output of the `kubectl get svc`. 
```

Comment:
Note that external access is enabled for both controller and segment store services.
It would be good to start it like this "When pravega is deployed with "external-access enabled", and we do a `kubectl get svc` the External IP for controller and segment store services appears truncated when using kubectl version `1.10.x.`
<Show the output>